### ISSUE TITLE
List of Blacklisted items.

### ISSUE THREAD

**Issue by xt9** [xt9/InWorldCrafting#1]
# Current blacklist
```
"astralsorcery", "logWood"
"astralsorcery", "astralsorcery:itemrockcrystalsimple"
"astralsorcery", "astralsorcery:itemcelestialcrystal"
"appliedenergistics2", "appliedenergistics2:material:1"
"appliedenergistics2", "appliedenergistics2:crystal_seed:*"
"actuallyadditions", "actuallyadditions:item_misc:23"
"actuallyadditions", "actuallyadditions:item_misc:24"
"deepmoblearning", "deepmoblearning:glitch_fragment"
```

If you know of an item that needs their own EntityItem for crafting logic or you have noticed some feature being broken as a result of adding a InWorldCrafting recipe then you can post about it here.

**Comment by xt9** [comment]
## Todo
- add Gold for extrautil2 (creates Demonsteel)
- add Dark Gem for EvilCraft (creates Dark Power Gem)
- add Redstone Dust for Flux Networks (creates Flux)
- add Purifying Bath Salts for Thaumcraft (creates Purifying Fluid)

### Change methods to have a manual override of the blacklist as an optional parameter.

**Comment by xt9** [comment]
No longer needed in v1.0, fixed in 16858492d08b253f14aecc25e86c8b837582bffa
